docs/help/account/billing.md

```diff
@@ -0,0 +1,44 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. Payment is by credit card only; we do not offer any alternatives at this time. We use Stripe to handle all payments, so we never store your bank details.
+
+## Invoices
+
+### Edit Your Billing Information
+
+You can edit the following billing information so that it appears on your upcoming invoices:
+
+- Company name
+- Address
+- VAT Number
+
+:::info
+Edited information will only appear on future invoices; it is not possible to edit the details of an already created invoice.
+:::
+
+### Invoices
+
+Every month, we send your invoice in PDF format to the email address linked to your account. You can specify an alternative email address to which they will be sent, for your accounting department, for example.
+
+On this page, you will also find a history of your invoices, available in PDF.
+
+### Taxes
+
+We apply taxes in accordance with current regulations. Three scenarios are possible:
+
+- you or your company reside outside of Europe: we do not apply VAT.
+- you or your company reside in Europe: we apply a 20% VAT unless you provide your VAT number in your billing details.
+
+## Payment Issues
+
+In the event of an issue with the charge on your credit card, Stripe will make several debit attempts in the following days. If, despite these attempts, payment remains unsuccessful, our Support team will contact you to assist in finding a solution.
```

Comment:
```suggestion
In the event of an issue with the charge on your credit card, our payment processor Stripe will make several debit attempts in the following days. If, despite these attempts, payment remains unsuccessful, our Support team will contact you to assist in finding a solution.
```

Reply:
To be exact, and as per the configuration of our Stripe account: Stripe will retry up to 4 times in 3 weeks to charge the credit card that was entered in the system. "Using machine learning, Smart Retries chooses the best times to retry failed payment attempts to increase the chance of successfully paying an invoice."

It could be worth mentioning that customers will get a warning 7 days upfront of the payment if their card is expired.

Sources:
https://stripe.com/docs/billing/revenue-recovery/smart-retries#smart-retries
![image](https://github.com/bump-sh/docs/assets/7700560/558a28f7-5300-4bfc-a9b5-f350097dca6f)

Reply:
@jayk-u you didn't respond to this comment, how do you want to proceed?